starters/vue3-xstate-css/src/router/index.ts

```diff
@@ -0,0 +1,27 @@
+import { createRouter, createWebHistory } from 'vue-router';
+
+const router = createRouter({
+	history: createWebHistory(import.meta.env.BASE_URL),
+	routes: [
+		{
+			path: '/',
+			name: 'home',
+			component: () => import('../views/HomeView.vue'),
```

Comment:
You can leave it as it is, but it is not suggested to "async load" the root of a website as you know that it will mostly always be loaded.

I know that for this size project it is ok, but it is a nice to have